lib/adapters/REST/endpoints/api-key.ts

```diff
@@ -1,20 +1,28 @@
 import type { AxiosInstance } from 'contentful-sdk-core'
 import * as raw from './raw'
-import { ApiKeyProps, CreateApiKeyProps } from '../../entities/api-key'
-import { CollectionProp, QueryParams, GetSpaceParams } from './common-types'
 import copy from 'fast-copy'
+import { GetSpaceParams, QueryParams } from '../../../common-types'
+import { ApiKeyProps, CreateApiKeyProps } from '../../../entities/api-key'
+import { CollectionProp } from '../../../common-types'
+import { RestEndpoint } from '../types'
 
-export const get = (http: AxiosInstance, params: GetSpaceParams & { apiKeyId: string }) => {
+export const get: RestEndpoint<'ApiKey', 'get'> = (
+  http: AxiosInstance,
+  params: GetSpaceParams & { apiKeyId: string }
+) => {
   return raw.get<ApiKeyProps>(http, `/spaces/${params.spaceId}/api_keys/${params.apiKeyId}`)
 }
 
-export const getMany = (http: AxiosInstance, params: GetSpaceParams & QueryParams) => {
+export const getMany: RestEndpoint<'ApiKey', 'getMany'> = (
+  http: AxiosInstance,
+  params: GetSpaceParams & QueryParams
+) => {
   return raw.get<CollectionProp<ApiKeyProps>>(http, `/spaces/${params.spaceId}/api_keys`, {
```

Comment:
Any reason we hardcode these URLs here instead of using functions to build these programmatically? For the `AppDefinition` we use something like `getBaseUrl`. Should we be consistent across the entities?

Reply:
Yes, the URL could be created programmatically.

I won't do it in this PR as I just copied the file: https://github.com/contentful/contentful-management.js/blob/master/lib/plain/endpoints/api-key.ts